srcalc/src/test/java/gov/va/med/srcalc/domain/model/DisplayNameComparatorTest.java

```diff
@@ -0,0 +1,29 @@
+package gov.va.med.srcalc.domain.model;
+
+import gov.va.med.srcalc.test.util.TestHelpers;
+
+import org.junit.Test;
+
+/**
+ * <p>Tests the {@link DisplayNameComparator} class.</p>
+ * 
+ * <p>Note that these test cases use BDD-style names "should {do something}".</p>
```

Comment:
You mean the test names? I had already written a couple tests like that, and I think this naming style has many benefits. I highly recommend reading [Introducing BDD](http://dannorth.net/introducing-bdd/) if you haven't already.